cmd/cli/kubectl-kyverno/jp/parse/parse.go

```diff
@@ -7,7 +7,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	gojmespath "github.com/jmespath/go-jmespath"
+	gojmespath "github.com/kyverno/go-jmespath"
```

Comment:
Presume this is an equal fork right now?

Reply:
Messed up my branch, should be clean now.

Reply:
Jmespath related PR is https://github.com/kyverno/kyverno/pull/7726 this is the same fork as before but i renamed the module so that we can `go install github.com/kyverno/kyverno/cmd/cli/kubectl-kyverno`.

The `replace` directive is not compatible with `go install`.

Reply:
Ok - I see.